### PR TITLE
toreless - well-known-uri

### LINKED ISSUE #66: Toreless - draft-5 section 3.1.1: clarify what the well-known URI is
````
298	   Additionally, certain pledge types may never attempt to discover a
299	   local domain registrar and may automatically bootstrap against a
300	   cloud registrar.

302	   The details of the URI are manufacturer specific, with BRSKI giving
303	   the example "brski-registrar.manufacturer.example.com".

Hmm. RFC8995 says:

performing a DNS lookup using a well-known URI such as "brski-registrar.manufacturer.example.com"

but i think thats wrong text because "brski-..." is not a URI given how it has no
scheme. Sentence should have been "DNS lookup using the host of the well-known URI".

I am actually not even sure how a BRSKI URI would look like. E.g.: when the
pledge just has a URI - how does it know that the protocol should be BRSKI - let's say
it does support multiple different enrolment protocols...  ???

https://brski-registrar.manufacturer.example.com/.well-known/brski

Is that a correct URI that fully identifies BRSKI ???
(i think it is...)
````

### PR DESCRIPTION
**Pull request by upros**
close #66

### CODE REVIEW

**Comment by upros** [comment]
close #66

File: anima-brski-cloud.md [hunk 1]
```diff
@@ -555,7 +555,7 @@ A "helpful" programmer might instead decide to redirect back to the manufacturer
 Instead, the VAR MUST return a 404 error if it cannot process the device.
 This will force the device to stop, timeout, and then try all mechanisms again.
 
-# Security Considerations
+# Secty Considerations
```
Comment (mcr) [inline anima-brski-cloud.md:558]:
```suggestion
# Security Considerations
```

File: anima-brski-cloud.md [hunk 2]
```diff
@@ -565,7 +565,7 @@ In addition to the dependency upon the MASA, the successful enrollment of a devi
 This internet accessible service may be operated by the manufacturer and/or by one or more value-added-resellers.
 All the considerations for operation of the MASA also apply to operation of the Cloud Registrar.
 
-## Security Updates for the Pledge
+## Secty Updates for the Pledge
```
Comment (mcr) [inline anima-brski-cloud.md:568]:
```suggestion
## Security Updates for the Pledge
```

**Review by mcr — CHANGES_REQUESTED** [review]
Some weird typos.

**Pull request merged**